lessons/intro/pyqt/index.md

```diff
@@ -16,19 +16,19 @@ Pokud budete používat svůj počítač, prosíme vás o instalaci již předem
 
 ### PyQt5
 
-Pokud máte Python 3.5 a jednu z platforem, pro které je připraven [wheel na PyPI](https://pypi.python.org/pypi/PyQt5), stačí udělat:
+Pokud máte Python 3.5 a jednu z platforem, pro které je připraven [wheel na PyPI](https://pypi.org/project/PyQt5/), stačí udělat:
```

Comment:
I'd wait with this. The page has a big red banner saying "This is a pre-production deployment of Warehouse."

Reply:
My idea was that the audience of MI-PYT can handle that. Also, we use pypi.org in different MI-PYT lessons as well.

Reply:
OK, it's your class :)